geoportailv3/static/js/notifyservice.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @fileoverview Provides a flash message service. That service helps
+ * displaying general messages.
+ */
+
+goog.provide('app.Notify');
+
+
+/**
+ * @typedef {function(string, string=)}
+ */
+app.Notify;
+
+
+/**
+ * @return {app.Notify} The notify function.
+ */
+app.notifyFactory = function() {
+
+  var container = angular.element('<div class="notify"></div>');
+  angular.element(document.body).append(container);
+
+  return notify;
+
+  /**
+   * @param {string} msg Message to show.
```

Comment:
Do we already have examples of templated translate strings (ie. with variables to be replaced)?